### PR TITLE
Update CMS 2012 data release announcement

### LINKED ISSUE #2228: templates: Update announcement of CMS 2012 release
- [x] Fix markdown:
    - [x] Superscript is broken
    - [x] Em dashes are broken
- [x] Add full text of final CMS announcement

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
- Fix markdown
- Add full text from original CMS update
- Closes #2228

Signed-off-by: Achintya Rao <achintya.rao@cern.ch>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
 <em>A collision event recorded by CMS in 2012 and available in the latest release of CMS Open Data. File for visualisation prepared by Tom McCauley</em></p>
+
+[LHC data are complicated and they are big.](https://cms.cern/news/lhc-data-be-made-public-open-access-initiative) The LHC began operations in 2010 and since then, under nominal conditions, intense bunches of protons collide inside the CMS detector up to 40 million times every second. CMS researchers have recorded petabytes of data from these collisions and have so far published [hundreds of scientific papers](http://cms.cern/news/cms-publishes-700th-research-paper) with these data. “Our data are an important element of the CMS Collaboration’s rich scientific legacy,” says CMS Spokesperson, Joel Butler. “We would like to ensure that they are not only preserved in the long run but are also available to the public, so that both CMS members and external researchers can re-examine them in the future. This is part of our [commitment to openness and long-term data preservation \[PDF\]](https://cms-docdb.cern.ch/cgi-bin/PublicDocDB/RetrieveFile?docid=6032&filename=CMSDataPolicy.pdf).”
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md]: The `PDF` part does not display nicely, I'll fix it while merging...

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md]: Noted. Sorry about that. Wanted to make sure the `<a>` tag got rendered correctly, but I guess this didn't work.

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md]: I can still push commits, if you want me to. Your call!

Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md]: OK, please push a change `\[PDF\]` to `(PDF)`.

Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/docs/observing-higgs-over-one-petabyte-new-cms-open-data/observing-higgs-over-one-petabyte-new-cms-open-data.md]: On it. I'll amend the previous commit so it's a single commit.

**Comment by RaoOfPhysics** [comment]
@tiborsimko: Fixed!

**Pull request merged**